Sources/MapboxMaps/Foundation/Extensions/Core/CacheManager.swift

```diff
@@ -1,6 +1,13 @@
 import Foundation
 
-extension CacheManager {
+extension MapboxCoreMaps.CacheManager {
```

Comment:
I know there was some convo around this, but does this need the full qualified name? Can we type alias it?